packages/core/content-manager/server/services/field-sizes.js

```diff
@@ -60,4 +60,26 @@ module.exports = () => ({
 
     return fieldSize;
   },
+  setFieldSize(type, size) {
+    if (!type) {
+      throw new Error('The type is required');
+    }
+
+    if (!size) {
+      throw new Error('The size is required');
+    }
+
+    fieldSizes[type] = size;
+  },
+  registerCustomFields() {
```

Comment:
The custom field has already been registered so I find the name a bit confusing. What do you think of something more along the lines of `setCustomFieldInputSizes`

Reply:
Yes I didn't realize that because I was working in the context of the field sizes plugin, but you're right